samples/kubeflow-tf/kubeflow-training-classification.py

```diff
@@ -13,75 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
+from pathlib import Path
 
 import kfp.dsl as dsl
 import kfp.gcp as gcp
-import datetime
 
-def dataflow_tf_transform_op(train_data: 'GcsUri', evaluation_data: 'GcsUri', schema: 'GcsUri[text/json]', project: 'GcpProject', preprocess_mode, preprocess_module: 'GcsUri[text/code/python]', transform_output: 'GcsUri[Directory]', step_name='preprocess'):
-    return dsl.ContainerOp(
-        name = step_name,
-        image = 'gcr.io/ml-pipeline/ml-pipeline-dataflow-tft:d3c4add0a95e930c70a330466d0923827784eb9a',
-        arguments = [
-            '--train', train_data,
-            '--eval', evaluation_data,
-            '--schema', schema,
-            '--project', project,
-            '--mode', preprocess_mode,
-            '--preprocessing-module', preprocess_module,
-            '--output', transform_output,
-        ],
-        file_outputs = {'transformed': '/output.txt'}
-    )
+from kfp.components import ComponentStore
 
+cs = ComponentStore()
+cs.local_search_paths.append(str(Path(__file__).parent.joinpath('../../components/'))) #local repo checkout path
+cs.url_search_prefixes.append('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/')
```

Comment:
These two lines are not needed for this sample. Although it still works with them, would it be better to move them to comments?

Reply:
You're right. The first line only helps with local development and the second line will only start working when the PR is merged.
I'll comment them out or remove them.